### PR TITLE
Implemented clipboard feature into models.py

### LINKED ISSUE #100: Feature Request: Implement clipboard feature into `show.py`
This file has already implemented [copy to clipboard feature](https://github.com/AmateursLeague/sneaky-package/blob/main/package/graph.py), integrate this feature into `models.py`.

**Requirement**:
- Implementation of clipboard feature into, `models.py`.
- Should only copy the contents into system's clipboard when executed with clipboard argument.

**Example**:
```python
from package.show import display
display("file", 1212, 1)
```
Here' the execution of models feature is same, but with extra argument being passed(i,e. `1`)

1. **Display feature**(**_What we already have!_**):
```python
from pacakge.show import display
display("test", 1212)
```
**Output**:
```bash
print("If it's visible, then it's good to go!")
```

2. **Copy to clipboard feature** (_**Required feature to be implemented**_)

```python
from pacakge.show import display
display("test", 1212, 1)
```
Should copy contents into system's clipboard without displaying the output in console.

The above feature explains the requirement of copy to clipboard feature inside `models.py`.

Share your ideas & thoughts on how you are planning to do this to get assigned!

### PR DESCRIPTION
**Pull request by yasheela-alla**
feat:  Clipboard Feature into models.py

### Related Issue

closes #100 

### Type of change

What sort of changes have you made:

- [x] New feature (non-breaking change which adds functionality)

### Checklist

- [x] My code follows the code style of this project.
- [x] I have followed the contribution guidelines
- [x] I have performed a self-review of my own code.
- [x] I have ensured my changes don't generate any new warnings or errors.
- [x] I have resolved all merge conflicts.

### CODE REVIEW

**Comment by yasheela-alla** [comment]
@yasheela-alla @ShashwatAgrawal20 Please review and let me know if you require any changes.

File: PyPi_Setup.md
Comment (ShashwatAgrawal20) [inline PyPi_Setup.md:1]: Not sure why you added this tho? Is that required?

File: package/models.py
Comment (ShashwatAgrawal20) [inline package/models.py:1]: @this-is-yaash not sure if this is what you required I suppose?

File: PyPi_Setup.md
Comment (yasheela-alla) [reply PyPi_Setup.md:1]: hey @ShashwatAgrawal20 That was my previous commit. I'm sure its not required here. will it create any issue to this PR?

**Comment by jindalpriyanshu101** [comment]
Any update @yasheela-alla ??

**Pull request closed without merging**